### ISSUE TITLE
Failed to resolve: com.fortysevendeg.swipelistview:swipelistview:1.0-SNAPSHOT

### ISSUE THREAD

**Issue by tomtharakan** [47deg/android-swipelistview#225]
Error:(38, 14) Failed to resolve: com.fortysevendeg.swipelistview:swipelistview:1.0-SNAPSHOT
<a href="openFile">Show in File</a><br><a href="open.dependency.in.project.structure">Show in Project Structure dialog</a>

**Comment by pablonajt** [comment]
@tomtharakan: If this helps, I have a colleague in India experiencing the same, looks to be an issue with him reaching the server from his site (the same build configuration works fine from me):

```
Error:A problem occurred configuring project ':jottmessenger'.
> Could not resolve all dependencies for configuration ':jottmessenger:_debugCompile'.
  > Could not resolve com.fortysevendeg.swipelistview:swipelistview:1.0-SNAPSHOT.
    Required by:
        jott-android:jottmessenger:unspecified
     > Unable to load Maven meta-data from http://clinker.47deg.com/nexus/content/groups/public/com/fortysevendeg/swipelistview/swipelistview/1.0-SNAPSHOT/maven-metadata.xml.
        > Could not GET 'http://clinker.47deg.com/nexus/content/groups/public/com/fortysevendeg/swipelistview/swipelistview/1.0-SNAPSHOT/maven-metadata.xml'.
           > Connection to http://clinker.47deg.com refused
```

**Comment by recena** [comment]
@pablonajt In fact, this issues is related to his internet provider.

@tomtharakan Could you tell me your current IP? I think that your IP is blocked.

**Comment by recena** [comment]
@pablonajt @tomtharakan Could your try it again?

**Comment by pablonajt** [comment]
@recena It works for my colleagues in India now. Thanks!

**Comment by recena** [comment]
@pablonajt Thanks for your feedback. Your ISP has been unblocked.

**Comment by nvhaiwork** [comment]
Hi @recena,
I have same issue, here is my ip address: 123.21.159.73

Can you help me ?

**Comment by recena** [comment]
@nvhaiwork Of course.

**Comment by nvhaiwork** [comment]
Sorry @recena but I still can't use your lib.
Here is log from Android studio:

Error:Failed to find: com.fortysevendeg.swipelistview:swipelistview:1.0-SNAPSHOT
Open File
Open in Project Structure dialog

Hope to see your reply, thank you very much.

**Comment by recena** [comment]
@nvhaiwork Can you access here? http://clinker.47deg.com/nexus/content/groups/public **AAR binary** is [here](http://clinker.47deg.com/nexus/service/local/repositories/sonatype-snapshots/content/com/github/rchugunov/swipelistview/1.0-SNAPSHOT/swipelistview-1.0-20140305.165728-2.aar)

**Comment by nvhaiwork** [comment]
@recena I think problem is my network provider.
Thanks for your support.
Nice day :P

**Comment by jgwiazdowski** [comment]
Error:A problem occurred configuring project ':APP'.
> Could not resolve all dependencies for configuration ':APP:_debugCompile'.
   > Could not find com.fortysevendeg.swipelistview:swipelistview:1.0-SNAPSHOT.
     Searched in the following locations:
         https://jcenter.bintray.com/com/fortysevendeg/swipelistview/swipelistview/1.0-SNAPSHOT/maven-metadata.xml
         https://jcenter.bintray.com/com/fortysevendeg/swipelistview/swipelistview/1.0-SNAPSHOT/swipelistview-1.0-SNAPSHOT.pom
         https://jcenter.bintray.com/com/fortysevendeg/swipelistview/swipelistview/1.0-SNAPSHOT/swipelistview-1.0-SNAPSHOT.aar
      /adt-bundle-linux-x86/sdk/extras/android/m2repository/com/fortysevendeg/swipelistview/swipelistview/1.0-SNAPSHOT/maven-metadata.xml
       Development/adt-bundle-linux-x86/sdk/extras/android/m2repository/com/fortysevendeg/swipelistview/swipelistview/1.0-SNAPSHOT/swipelistview-1.0-SNAPSHOT.pom
       Development/adt-bundle-linux-x86/sdk/extras/android/m2repository/com/fortysevendeg/swipelistview/swipelistview/1.0-SNAPSHOT/swipelistview-1.0-SNAPSHOT.aar
    Development/adt-bundle-linux-x86/sdk/extras/google/m2repository/com/fortysevendeg/swipelistview/swipelistview/1.0-SNAPSHOT/maven-metadata.xml
       Development/adt-bundle-linux-x86/sdk/extras/google/m2repository/com/fortysevendeg/swipelistview/swipelistview/1.0-SNAPSHOT/swipelistview-1.0-SNAPSHOT.pom
        Development/adt-bundle-linux-x86/sdk/extras/google/m2repository/com/fortysevendeg/swipelistview/swipelistview/1.0-SNAPSHOT/swipelistview-1.0-SNAPSHOT.aar

so you're saying that issue with network provider? O.O

**Comment by oguzozcan** [comment]
I have the same issue as well.. I can't access to AAR binary. Could you please help me?

My IP is: 46.1.40.29

**Comment by niksneel4u** [comment]
I have same issue, here is my ip address: 124.125.242.132
plzz help me....
thnaxxxxx in advance

**Comment by xajker** [comment]
I have the same problem, my ip 77.47.198.199

**Comment by Sunno** [comment]
It seems repo is down, I'm having same issue, I can't compile my project because of that... :/

**Comment by mjdeverill** [comment]
Yes, repo is down. Trying to access from Seville, Spain. Any way other way of linking? I mean, downloading the library and linking offline? Thanks in advance

**Comment by Sunno** [comment]
I guess you can try downloading the zip file, unzip it and include as a module in your project.

**Comment by mjdeverill** [comment]
Thanks Sunno, I'll try!

**Comment by xajker** [comment]
If somebody find a solution let us know, please.

**Comment by mjdeverill** [comment]
I've downloaded the code, and this one: http://nineoldandroids.com/ and added both packages to my project. It built.

**Comment by Sunno** [comment]
Actually I haven't been able to build (I hate gradle and their criptic messages), I imported module but it never finishes adding it since it returns a "Gradle DSL method not found: 'compile()'".

**Comment by crucibond** [comment]
I am facing a similar situation and have the same error message. Should i just download the code and include it in my project.

**Comment by mmartin101** [comment]
For those of us still using this project we need to find a way to keep this going. Maybe we can submit this to jcenter/bintray?

**Comment by Aijaz-Ali-Shah** [comment]
Hi @recena,
I have same issue, here is my ip address: 10.105.19.106

Can you please help me ?
